### PR TITLE
Add autogenerated Openrocket CSVs on push

### PR DESCRIPTION
**Pull request by cowsed**
Adds a github action that takes the arguments:
- URL to download the Openrocket JAR from
- Openrocket Version Name
- Openrocket Filename

Then does a bunch of setup

Then exports all simulations to CSVs with all available data in SI units

Then commits the CSVs folder back to the repository

### CODE REVIEW

File: scripts/export_all.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+with orhelper.OpenRocketInstance(jar_path=orJarPath, log_level='ALL') as instance:
+    orh = orhelper.Helper(instance)
+    orp = jpype.JPackage("net").sf.openrocket
+
+    FileOutputStream = jpype.JClass("java.io.FileOutputStream")
+    CSVExport = orp.file.CSVExport
+
+    doc = orh.load_doc(rocketFile)
+    numSims = doc.getSimulationCount()
```
Comment (AarC10) [inline scripts/export_all.py]: snake_case

File: scripts/export_all.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        branch = sim.getSimulatedData().getBranch(0)
+        if sim.getSimulatedData().getBranchCount() > 1:
+            print(f"Only using branch 1 of {
+                  sim.getSimulatedData().getBranchCount()}")
+
+        flight_data_types = branch.getTypes()
+        units = [datatype.getUnitGroup().getSIUnit()
+                 for datatype in flight_data_types]
+
+        path = os.path.join(outputDir, simName+'.csv')
```
Comment (AarC10) [inline scripts/export_all.py]: Space the +

**Review by AarC10 — APPROVED** [review]

File: scripts/export_all.py (hunk 1 above)
Comment (cowsed) [reply scripts/export_all.py]: the opps hate to see me try to sneak shitty python into one of our repos

Comment (cowsed) [reply scripts/export_all.py]: doen

File: scripts/export_all.py (hunk 2 above)
Comment (cowsed) [reply scripts/export_all.py]: wam bam all done

Comment (cowsed) [reply scripts/export_all.py]: sike i lied wrong remote

**Pull request closed without merging**